files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -105,6 +105,7 @@
 default['private_chef']['rabbitmq']['nodename'] = 'rabbit@localhost'
 default['private_chef']['rabbitmq']['vip'] = '127.0.0.1'
 default['private_chef']['rabbitmq']['consumer_id'] = 'hotsauce'
+default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games"
```

Comment:
We can probably pare this down to `/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin`